src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;

```

Comment:
The `FruitTransaction` class should be placed in its own file within the `model` package, which is correctly done here. However, it's important to ensure that the rest of the codebase follows the same organizational structure, with classes being placed in appropriate packages.